src/pages/App.js

```diff
@@ -63,7 +64,32 @@ export default function App() {
       />
       <button type="submit" onClick={handleSubmit}>Search</button>
       <p>
-        {foods.map((elem) => <li>{elem.description}</li>)}
+        {foods.slice(10, 30).map((elem) => {
+          if (elem.description.toLowerCase() === food.toLowerCase()) {
+            return null;// no need to show it in the page
+          }
+          if (!set.has(elem.description.toLowerCase())) {
```

Comment:
I think this would be a bit cleaner with the `return null;` as an early return. That is, instead of checking for absence from the set, you can check for existence in the set and return null, then return the element afterwards.